vizro-core/src/vizro/models/_components/form/date_picker.py

```diff
@@ -27,7 +26,7 @@ class DatePicker(VizroBaseModel):
     [`dmc.DateRangePicker`](https://www.dash-mantine-components.com/components/datepicker#daterangepicker).
 
     Args:
-        type (Literal["date_picker"]): Defaults to `"date_picker"`.
+        type (Literal["default", "range", "multiple"]): Defaults to A single date picker allowing the selection of one date.
```

Comment:
I understand the confusion. This argument isn't the same as the `type` argument inside the `dmc` component. It's actually an argument that all our pydantic models have to differentiate their types. So, we shouldn't change it. If we ever want to introduce the `type` argument from the `dmc` component, we should add it as a new argument with a different name. But for now, let's leave it as is and not introduce it yet 👍 

```suggestion
        type (Literal["date_picker"]): Defaults to `"date_picker"`.
```